components/sup/src/event.rs

```diff
@@ -16,6 +16,8 @@ mod stream;
 mod types;
 
 pub(crate) use self::types::ServiceMetadata;
+use prost_types::Duration as prost_duration;
+
```

Comment:
Extra newline. If we keep our imports together `rustfmt` will automatically sort them.